test/acceptance/test/add_tests.go

```diff
@@ -990,17 +1044,11 @@ var _ = Describe("Weave GitOps App Add Tests", func() {
 		})
 
 		By("When I rerun wego gitops install", func() {
-			reinstallOutput, _ = runCommandAndReturnStringOutput(WEGO_BIN_PATH + " gitops install")
+			_ = runCommandPassThrough([]string{}, "sh", "-c", fmt.Sprintf("%s gitops install", WEGO_BIN_PATH))
 		})
 
 		By("Then I should not see any errors", func() {
-			Eventually(reinstallOutput).Should(ContainSubstring("► installing components in " + WEGO_DEFAULT_NAMESPACE + " namespace"))
-			Eventually(reinstallOutput).Should(ContainSubstring("✔ image-reflector-controller: deployment ready"))
-			Eventually(reinstallOutput).Should(ContainSubstring("✔ image-automation-controller: deployment ready"))
-			Eventually(reinstallOutput).Should(ContainSubstring("✔ source-controller: deployment ready"))
-			Eventually(reinstallOutput).Should(ContainSubstring("✔ kustomize-controller: deployment ready"))
-			Eventually(reinstallOutput).Should(ContainSubstring("✔ helm-controller: deployment ready"))
-			Eventually(reinstallOutput).Should(ContainSubstring("✔ notification-controller: deployment ready"))
```

Comment:
Does this have something to do with `get commits`, or is it a fix for something else?

Reply:
This a solution to the flakiness we've been seeing in one of the acceptance tests. Instead of verifying the output of `wego gitops install` rerun, I'm making sure the controller pods are in ready state. Hopefully, we've see failures on this test case from now